### PR TITLE
Fabric Block render layers v1

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
New features :-
* Allows setting block render layers much more easily
* Allows setting block render layers for vanilla blocks that use the default one

### CODE REVIEW

File: fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java [hunk 1]
```diff
@@ -14,16 +14,22 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.impl.item.group;
```
Comment (hYdos) [inline fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java:17]: whats with the different package

Comment (BoogieMonster1O1) [reply fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java:17]: That was always that way

Comment (BoogieMonster1O1) [reply fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java:17]: I only removed it from `unfinished`

Comment (hYdos) [reply fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java:17]: ah ok

**Comment by shedaniel** [comment]
Do we need this? I don't think we would need this, and for vanilla blocks mods should just mixin as vanilla blocks that have a different block layer override that method so this will not be called.

We don't use to have this API before 1.15 when they switched to a map.

**Comment by BoogieMonster1O1** [comment]
Sure

**Pull request closed without merging**